src/index.html

```diff
@@ -11,7 +11,180 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form">
+      <fieldset class="form_info form">
+        <legend>Personal information</legend>
+        <div class="info_surname info">
+          <label for="surname">Surname:
```

Comment:
```suggestion
          <label for="surname">
            Surname:
```

Reply:
Don't forget to wrap inner content of tag to new line if there are no closing tag after it

Reply:
Good
```
<h1>Hello</h1>
```

Good
```
<h1>
  Hello
</h1>
```

Bad
```
<h1>Hello
</h1>
```

Reply:
Please fix it everywhere